Fixes make with -j, otherwise scotch.h may not be generated by the time make
tries to compile library_graph_diam.o et al.

Reported upstream at
https://gforge.inria.fr/tracker/index.php?func=detail&aid=21651&group_id=248&atid=1081

diff --git a/src/libscotch/Makefile b/src/libscotch/Makefile
index 9898894..b0ee14c 100644
--- a/src/libscotch/Makefile
+++ b/src/libscotch/Makefile
@@ -2229,6 +2229,28 @@ library_graph_color_f$(OBJ)	:	library_graph_color_f.c			\
 					common.h				\
 					scotch.h
 
+library_graph_diam$(OBJ)	:	library_graph_diam.c			\
+					module.h				\
+					common.h				\
+					graph.h					\
+					scotch.h
+
+library_graph_diam_f$(OBJ)	:	library_graph_diam.c			\
+					module.h				\
+					common.h				\
+					scotch.h
+
+library_graph_induce$(OBJ)	:	library_graph_diam.c			\
+					module.h				\
+					common.h				\
+					graph.h					\
+					scotch.h
+
+library_graph_induce_f$(OBJ)	:	library_graph_diam.c			\
+					module.h				\
+					common.h				\
+					scotch.h
+
 library_graph_io_chac$(OBJ)	:	library_graph_io_chac.c			\
 					module.h				\
 					common.h				\
